proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -11814,56 +11814,157 @@ function longAreRepeatedProbe( test )
 function longAllAreRepeated( test )
 {
 
+  test.case = 'empty array';
   var got = _.longAllAreRepeated( [] );
-  test.identical( got, true );
+  var expected = true;
+  test.identical( got, expected );
 
+  test.case = 'identical numbers of array';
   var got = _.longAllAreRepeated( [ 1, 1 ] );
-  test.identical( got, true );
+  var expected = true;
+  test.identical( got, expected );
 
+  test.case = 'single number of array';
   var got = _.longAllAreRepeated( [ 1 ] );
-  test.identical( got, false );
+  var expected = false;
+  test.identical( got, expected );
 
+  test.case = 'some numbers of array are identical';
   var got = _.longAllAreRepeated( [ 1, 2, 2 ] );
-  test.identical( got, false );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
 
+  test.case = 'identical strings of array';
+  var got = _.longAllAreRepeated( [ 'string', 'string' ] );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'some strings of array are identical';
+  var got = _.longAllAreRepeated( [ 'string', 'string', 'diff string' ] );
```

Comment:
Routine `longAllAreRepeated` uses callback `onEvaluate`. Please, add test cases with [`evaluator` and `equalizer`](https://github.com/dmvict/wTools/blob/master/doc/ukr/tutorial/ConventionCallbecksInArray.md) callbacks.